api/rhsm-subscriptions-api-spec.yaml

```diff
@@ -59,7 +63,7 @@ paths:
           content:
             application/vnd.api+json:
               schema:
-                $ref: "#/components/schemas/TallyReport"
+                $ref: "#/components/schemas/InsightsResult"
```

Comment:
I think this should stay as `TallyReport`; see following comment.

Reply:
I was thinking we'd name it InsightsResult because we could drop other things into it using Openapi [polymorphism](https://swagger.io/docs/specification/data-models/inheritance-and-polymorphism/).  I.e. the `data` section of an InsightsResult can contain a TallyReport or a SomethingElseReport.  If you don't think we need that level of extensibility, I'll flip it back.